<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewPostWidget</class>
 <widget class="QWidget" name="ViewPostWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="usersLayout"/>
   </item>
   <item>
    <widget class="QTextEdit" name="mpEdit"/>
   </item>
   <item>
    <layout class="QGridLayout" name="filesLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
